server/swap/swap.go

```diff
@@ -1035,15 +1071,15 @@ func (s *Swapper) processBlock(block *blockNotification) {
 			}
 			// If the maker has broadcast their transaction, the taker's broadcast
 			// timeout starts once the maker's swap has SwapConf confs.
-			s.tryConfirmSwap(match.makerStatus)
+			s.tryConfirmSwap(match.makerStatus, block.time)
 		case order.TakerSwapCast:
 			if match.takerStatus.swapAsset != block.assetID {
 				break statusSwitch
 			}
 			// If the taker has broadcast their transaction, the maker's broadcast
 			// timeout (for redemption) starts once the maker's swap has SwapConf
 			// confs.
-			s.tryConfirmSwap(match.takerStatus)
```

Comment:
BTW, it occurred to me that I don't see much point in the RedeemStatus stuff, where we check the confirmation count on the redeem transactions.  Planning to investigate changes w.r.t. this as there's not much reason to wait for redeem confirms to stop tracking the match.  This is closely related to the issue of the [maker's ack of the taker's redeem](https://github.com/decred/dcrdex/issues/692), so I'm going to consider that at the same time and how these things interact w.r.t. timing (e.g. I think we've seen matches removed before the maker's final pointless redeem ack).